InterfaceComputer/Computer.java

```diff
@@ -0,0 +1,116 @@
+package InterfaceComputer;
+
+
+public class Computer implements NewComponents {
+
+    private String modelMonitor;
+    private String cpu;
+    private String gpu;
+    private int ram;
```

Comment:
What do you think they can be a part of phone? or pad?
Can we extract common parts to abstract class?

Reply:
and try to use classes instead Strings. for example: create class CPU with some parameters: model, caches and etc

Reply:
think about interface names

Reply:
Comments have been corrected